addons/binding/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueSensorHandler.java

```diff
@@ -261,7 +268,14 @@ public void channelLinked(ChannelUID channelUID) {
     @Override
     public void onSensorRemoved(@Nullable HueBridge bridge, FullSensor sensor) {
         if (sensor.getId().equals(sensorId)) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "offline.sensor-removed");
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "offline.sensor-not-reachable");
+        }
+    }
+
+    @Override
+    public void onSensorGone(@Nullable HueBridge bridge, FullSensor sensor) {
+        if (sensor.getId().equals(sensorId)) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.GONE, "offline.conf-error-wrong-sensor-id");
```

Comment:
Shouldn't the messages be prefixed with `@text/`?
```suggestion
            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.GONE, "@text/offline.conf-error-wrong-sensor-id");
```